bookshelf/app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,46 @@
+# Talentica - bootstrap project
+# Copyright   Ashwini Kumar
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+class RegistrationsController < Devise::RegistrationsController
+  # registration of new user by overriding native devise
+  before_filter :authenticate_user!
+
+  def user_email
+    @response = User.email_validator params["email"]
```

Comment:
Hi @ashwinimnnit, did you merge bs_develop into this branch because I remember renaming the 'email_validator' method.